openassessment/xblock/static/js/src/studio/oa_container_item.js

```diff
@@ -76,11 +76,44 @@ OpenAssessment.ItemUtilities = {
  OpenAssessment.Prompt
  **/
 OpenAssessment.Prompt = function(element, notifier) {
+    if (this.tinyMCEEnabled) {
+        element = this.attachWysiwygToPrompt(element);
+    }
     this.element = element;
     this.notifier = notifier;
 };
 
 OpenAssessment.Prompt.prototype = {
+    tinyMCEEnabled: window.tinyMCE !== undefined,
+
+    /**
+     Attach Wysiwyg editor to the textarea field.
+
+     Args:
+     el (OpenAssessment.Container): The container that the prompt is a member of.
+
+     Returns:
+     Updated OpenAssessment.Container
+
+     **/
+    attachWysiwygToPrompt: function(el) {
+        var elId = $(el).find('textarea').attr('id');
+        if (!elId) {
+            /* jshint undef:false */
+            var textarea = $(el).find('textarea');
+            var text = $(textarea).val();
+            var type = $(textarea).data('type');
+            if (text && (type === 'text')) {
+                text = _.escape(text).replace(/(?:\r\n|\r|\n)/g, '<br />');
+                $(textarea).val(text);
+            }
+            var newElId = Date.now() + '-textarea-' + (Math.random() * 100);
+            $(textarea).attr('id', newElId).tinymce(tinymceCfg());
+            window.tinyMCE.get(newElId).focus();
```

Comment:
this line is causing the error on remove of prompt. Works fine when creating the prompt. Probably just need to add a check that the element is not null before calling focus on it

Reply:
you should probably add a test for removing the prompt as well as adding one